doc/Process_Console.md

```diff
@@ -0,0 +1,588 @@
+ Process Console
+ ===============
+ 
+ `process_console` is a capsule that implements a text console over the UART that allows
+ a terminal to inspect and control userspace processes.
+
+ Protocol
+ --------
+
+ This module provides a simple text-based console to inspect and control
+ which processes are running. The console has eleven commands:
+  - `help` - prints the available commands and arguments
+  - `status` - prints the current system status
+  - `list` - lists the current processes with their IDs and running state
+  - `stop n` - stops the process with name n
+  - `start n` - starts the stopped process with name n
+  - `fault n` - forces the process with name n into a fault state
+  - `terminate n` - terminates the running process with name n, moving to the Terminated state
+  - `boot n` - tries to boot a Terminated process with name n
```

Comment:
What does `boot` mean? Restart?

Reply:
It means restart, I've changed the description.